nearcore/src/config.rs

```diff
@@ -306,6 +306,8 @@ pub struct Config {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub save_trie_changes: Option<bool>,
     pub log_summary_style: LogSummaryStyle,
+    /// Makes logging of BlocksInfo span 1 line instead of being unlimited.
```

Comment:
Shouldn't the comment be the other way around? (Makes log unlimited instead of single line)